### PR TITLE
Added NSLinguisticTagger to detect from language.

### PR DESCRIPTION
**Pull request by JamalK**
Made a String Extension that returns the Language? if detected.

### CODE REVIEW

File: Polyglot/StringExtension.swift [hunk 1]
```diff
@@ -29,4 +29,16 @@ public extension String {
         urlQueryAllowedCharacterSet.removeCharactersInString("&=?+")
         return self.stringByAddingPercentEncodingWithAllowedCharacters(urlQueryAllowedCharacterSet)
     }
+    
+    public var language: Language? {
+        if (countElements(self) > 1) { //Prevent Index Out of Bounds in NSLinguisticTagger
```
Comment (ayanonagon) [inline Polyglot/StringExtension.swift]: Is this supposed to be `> 1` or `> 0`?

**Comment by ayanonagon** [comment]
Thanks for the PR! Just left one question. Also, it would be fantastic if you could add a basic test or two for `language` in https://github.com/ayanonagon/Polyglot/blob/master/PolyglotTests/StringExtensionTests.swift

Let me know if you have any questions around that. Thanks again for this PR. Super awesome. :octocat: :+1: :+1:

File: Polyglot/StringExtension.swift (hunk 1 above)
Comment (JamalK) [inline Polyglot/StringExtension.swift]: You're right, it should be ```> 0```.

**Comment by ayanonagon** [comment]
Looks great. Thank you for this pull-request (and the first feature addition to Polyglot)! I think it’s a really useful feature to have. :smile_cat:

**Comment by JamalK** [comment]
You're welcome :+1: I really like the idea behind Polyglot, and wanted to contribute.

**Pull request merged**